From 1a172b406add619802db67b04f3cab5cd6d39591 Mon Sep 17 00:00:00 2001
From: Iordan Iordanov <iiordanov@gmail.com>
Date: Tue, 23 Mar 2021 15:35:32 -0400
Subject: [PATCH 1/5] Disable sync of audio playback and record volume

---
 src/channel-main.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/channel-main.c b/src/channel-main.c
index 7f71258..a94bddc 100644
--- a/src/channel-main.c
+++ b/src/channel-main.c
@@ -2089,8 +2089,8 @@ static void main_agent_handle_msg(SpiceChannel *channel,
             c->agent_display_config_sent = true;
         }
 
-        agent_sync_audio_playback(self);
-        agent_sync_audio_record(self);
+        //agent_sync_audio_playback(self);
+        //agent_sync_audio_record(self);
 
         agent_max_clipboard(self);
 
-- 
2.25.1

