From e5fe929df81b21d525323dde04d8db837c3be2e2 Mon Sep 17 00:00:00 2001
From: Iordan Iordanov <iiordanov@gmail.com>
Date: Wed, 24 Mar 2021 19:10:02 -0400
Subject: [PATCH 3/3] Disable notify volume and notify mute for playback and
 record

---
 src/spice-gstaudio.c | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/src/spice-gstaudio.c b/src/spice-gstaudio.c
index 38e3d73..caf5217 100644
--- a/src/spice-gstaudio.c
+++ b/src/spice-gstaudio.c
@@ -520,10 +520,10 @@ static gboolean connect_channel(SpiceAudio *audio, SpiceChannel *channel)
                                       G_CALLBACK(playback_data), gstaudio, 0);
         spice_g_signal_connect_object(channel, "playback-stop",
                                       G_CALLBACK(playback_stop), gstaudio, G_CONNECT_SWAPPED);
-        spice_g_signal_connect_object(channel, "notify::volume",
-                                      G_CALLBACK(playback_volume_changed), gstaudio, 0);
-        spice_g_signal_connect_object(channel, "notify::mute",
-                                      G_CALLBACK(playback_mute_changed), gstaudio, 0);
+        //spice_g_signal_connect_object(channel, "notify::volume",
+        //                              G_CALLBACK(playback_volume_changed), gstaudio, 0);
+        //spice_g_signal_connect_object(channel, "notify::mute",
+        //                              G_CALLBACK(playback_mute_changed), gstaudio, 0);
 
         return TRUE;
     }
@@ -538,10 +538,10 @@ static gboolean connect_channel(SpiceAudio *audio, SpiceChannel *channel)
                                       G_CALLBACK(record_start), gstaudio, 0);
         spice_g_signal_connect_object(channel, "record-stop",
                                       G_CALLBACK(record_stop), gstaudio, G_CONNECT_SWAPPED);
-        spice_g_signal_connect_object(channel, "notify::volume",
-                                      G_CALLBACK(record_volume_changed), gstaudio, 0);
-        spice_g_signal_connect_object(channel, "notify::mute",
-                                      G_CALLBACK(record_mute_changed), gstaudio, 0);
+        //spice_g_signal_connect_object(channel, "notify::volume",
+        //                              G_CALLBACK(record_volume_changed), gstaudio, 0);
+        //spice_g_signal_connect_object(channel, "notify::mute",
+        //                              G_CALLBACK(record_mute_changed), gstaudio, 0);
 
         return TRUE;
     }
-- 
2.17.1

