From 69086befd554f8bb7089c7a2cb6f2ef1ce701c74 Mon Sep 17 00:00:00 2001
From: Iordan Iordanov <iiordanov@gmail.com>
Date: Wed, 24 Mar 2021 19:10:02 -0400
Subject: [PATCH 3/5] Disable notify volume and notify mute for playback and
 record

---
 src/spice-gstaudio.c | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/src/spice-gstaudio.c b/src/spice-gstaudio.c
index e472baf..fea057b 100644
--- a/src/spice-gstaudio.c
+++ b/src/spice-gstaudio.c
@@ -529,10 +529,10 @@ static gboolean connect_channel(SpiceAudio *audio, SpiceChannel *channel)
                                       G_CALLBACK(playback_data), gstaudio, 0);
         spice_g_signal_connect_object(channel, "playback-stop",
                                       G_CALLBACK(playback_stop), gstaudio, G_CONNECT_SWAPPED);
-        spice_g_signal_connect_object(channel, "notify::volume",
-                                      G_CALLBACK(playback_volume_changed), gstaudio, 0);
-        spice_g_signal_connect_object(channel, "notify::mute",
-                                      G_CALLBACK(playback_mute_changed), gstaudio, 0);
+        //spice_g_signal_connect_object(channel, "notify::volume",
+        //                              G_CALLBACK(playback_volume_changed), gstaudio, 0);
+        //spice_g_signal_connect_object(channel, "notify::mute",
+        //                              G_CALLBACK(playback_mute_changed), gstaudio, 0);
 
         return TRUE;
     }
@@ -547,10 +547,10 @@ static gboolean connect_channel(SpiceAudio *audio, SpiceChannel *channel)
                                       G_CALLBACK(record_start), gstaudio, 0);
         spice_g_signal_connect_object(channel, "record-stop",
                                       G_CALLBACK(record_stop), gstaudio, G_CONNECT_SWAPPED);
-        spice_g_signal_connect_object(channel, "notify::volume",
-                                      G_CALLBACK(record_volume_changed), gstaudio, 0);
-        spice_g_signal_connect_object(channel, "notify::mute",
-                                      G_CALLBACK(record_mute_changed), gstaudio, 0);
+        //spice_g_signal_connect_object(channel, "notify::volume",
+        //                              G_CALLBACK(record_volume_changed), gstaudio, 0);
+        //spice_g_signal_connect_object(channel, "notify::mute",
+        //                              G_CALLBACK(record_mute_changed), gstaudio, 0);
 
         return TRUE;
     }
-- 
2.25.1

